.github/workflows/test-cluster-set-up.yml

```diff
@@ -0,0 +1,70 @@
+name: Create testing cluster
+
+# Make sure to pass the distribution type and security feature as parameters
+# { 
+#  "distribution": "RPM/DEB/TAR",
+#  "security": "enable/disable"
+# }
+
+on:
+  repository_dispatch:
+    types: [test-cluster-set-up]
```

Comment:
once we verify this can work well, let's schedule it with daily schedule

Reply:
Sure